cns/multitenantcontroller/mockclients/Makefile

```diff
@@ -5,7 +5,7 @@ MOCKGEN = $(TOOLS_BIN_DIR)/mockgen
 
 generate: $(MOCKGEN) ## Generate mock clients
 	$(MOCKGEN) -source=$(REPO_ROOT)/cns/multitenantcontroller/multitenantoperator/multitenantcrdreconciler.go -package=mockclients cnsRESTservice > cnsrestservice_generated.go
-	$(MOCKGEN) -source=$(REPO_ROOT)/vendor/sigs.k8s.io/controller-runtime/pkg/client/interfaces.go -package=mockclients Client > kubeclient_generated.go
+	$(MOCKGEN) -source=$(REPO_ROOT)/vendor/sigs.k8s.io/controller-runtime/pkg/client/interfaces.go -package=mockclients Client,SubResourceWriter > kubeclient_generated.go
```

Comment:
can you provide some context on this?

Reply:
yes, previously our tests only used a mock of the `client.Client` but now we also need a mock of the `client.Status()`'s `client.SubResourceWriter`. So this is just being explicit about which symbols we want to generate.

Reply:
sounds good ty